cmd/validate.go

```diff
@@ -66,6 +79,60 @@ this command.
 	},
 }
 
+func validate(ctx context.Context, entity interface{}, kongClient *kong.Client, entityType string) string {
+	_, err := performValidate(ctx, kongClient, entity, entityType)
+	output := ""
+	if err != nil {
+		output = color.New(color.FgRed, color.Bold).Sprintf("%s: %v", entityType, err)
```

Comment:
use the `cprint` package, which wraps the color package with some additional behavior.